pootle/apps/accounts/migrations/0003_remove_pootleprofile_id.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def fix_accounts_alt_src_langs(apps, schema_editor):
+    cursor = schema_editor.connection.cursor()
+
+    # check its mysql - should probs check its not too old
+    if not hasattr(cursor.db, "mysql_version"):
```

Comment:
The connection is using the settings, so it is safe. IMHO it is preferable to avoid checking the connection using these hackeries.